apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -62,4 +66,37 @@ const getAuthenticatedUser = createAsyncThunk<
 	return await authApi.getAuthenticatedUser();
 });
 
-export { getAuthenticatedUser, logOut, signIn, signUp };
+const requestResetPassword = createAsyncThunk<
+	boolean,
+	EmailDto,
+	AsyncThunkConfig
+>(`${sliceName}/send-reset-password-link`, async (emailPayload, { extra }) => {
+	const { authApi, notification } = extra;
+
+	notification.success(NotificationMessage.LINK_SENT);
+
+	return await authApi.requestResetPassword(emailPayload);
```

Comment:
You send a notification and only after that send a request to Backend. What if the Backend fails to respond?